### ISSUE TITLE
Yuki

### ISSUE THREAD

**Issue by Erwanrevirand45** [ZukaBri3k/CommandInvite#1]
import tensorflow as tf
from tensorflow.keras import layers, models

# Charger les données d'entraînement (par exemple, depuis TensorFlow datasets)
(train_images, train_labels), (test_images, test_labels) = tf.keras.datasets.mnist.load_data()

# Prétraiter les données
train_images = train_images / 255.0
test_images = test_images / 255.0

# Construire le modèle
model = models.Sequential([
    layers.Flatten(input_shape=(28, 28)),  # Aplatir l'image en une seule dimension
    layers.Dense(128, activation='relu'),  # Couche cachée avec 128 neurones et activation relu
    layers.Dense(10, activation='softmax') # Couche de sortie avec 10 neurones (pour les 10 classes) et activation softmax
])

# Compiler le modèle
model.compile(optimizer='adam',
              loss='sparse_categorical_crossentropy',
              metrics=['accuracy'])

# Entraîner le modèle
model.fit(train_images, train_labels, epochs=10, validation_data=(test_images, test_labels))

# Évaluer le modèle
test_loss, test_acc = model.evaluate(test_images, test_labels)
print('Test accuracy:', test_acc)

**Comment by Erwanrevirand45** [comment]
Comment mettre en français
